plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/util/CassandraCqlUtils.java

```diff
@@ -116,8 +116,8 @@ public static Select selectFrom(CassandraTableHandle tableHandle, List<Cassandra
 
     public static SelectFrom from(CassandraTableHandle tableHandle)
```

Comment:
WDYT - Instead of passing `CassandraTableHandle` can we pass `CassandraNamedRelationHandle`. 

Just an idea - It could be implemented as a follow up or a preparatory commit.